CMakeLists.txt

```diff
@@ -90,6 +90,11 @@ if (ONEDPL_USE_PARALLEL_POLICIES)
                     target_compile_definitions(oneDPL INTERFACE TBB_USE_GLIBCXX_VERSION=70300)
                 endif()
             endif()
+            if(CMAKE_CXX_STANDARD GREATER_EQUAL 17)
+                if(_onedpl_gxx_version AND _onedpl_gxx_version GREATER_EQUAL 80100 OR NOT _onedpl_gxx_version)
```

Comment:
`_onedpl_gxx_version` is initiated under `if (CMAKE_CXX_COMPILER_ID STREQUAL Clang)`, so it will properly work only if compiler is Clang. Moreover `_onedpl_gxx_version` contains post processed value (`.` are replaced with `0`). Feel free to reconsider to existing approach with g++ version detection and using it in `TBB_USE_GLIBCXX_VERSION`.

Consider: detect g++ version somewhere in the beginning of the configuration and then use it for TBB_USE_GLIBCXX_VERSION and for ranges api enabling.